Makefile

```
@@ -41,6 +44,7 @@ test_yes_against_bsd_yes:
 
 test_executables_against_their_bsd_counterparts:
 	make test_cat_against_bsd_cat
+	test_grep_against_bsd_grep
```

```suggestion
	make test_grep_against_bsd_grep
```

(Would this run otherwise?)